applications/vanilla/settings/class.hooks.php

```diff
@@ -697,8 +697,11 @@ public function profileController_addProfileTabs_handler($sender) {
             $discussionsLabel = sprite('SpDiscussions').' '.t('Discussions');
             $commentsLabel = sprite('SpComments').' '.t('Comments');
             if (c('Vanilla.Profile.ShowCounts', true)) {
-                $discussionsLabel .= '<span class="Aside">'.countString(getValueR('User.CountDiscussions', $sender, null), "/profile/count/discussions?userid=$userID").'</span>';
-                $commentsLabel .= '<span class="Aside">'.countString(getValueR('User.CountComments', $sender, null), "/profile/count/comments?userid=$userID").'</span>';
+                $discussionsCount = getValueR('User.CountDiscussions', $sender, null);
+                $commentsCount = getValueR('User.CountComments', $sender, null);
+
+                $discussionsLabel .= '<span class="Aside">'.countString(bigPlural($discussionsCount, '%s discussions'), "/profile/count/discussions?userid=$userID").'</span>';
```

Comment:
If I remember correctly `bigPlural` function expects to get "single" variant of string `'%s discussions'` -> `'%s discussion'`